lib/html2rss/configs/generator/file_creator.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module Html2rss
+  module Configs
+    module Generator
+      class FileCreator
+        RSPEC_TEMPLATE = <<~RSPEC
+          # frozen_string_literal: true
+
+          RSpec.describe '%<relative_yml_file_path>s' do
+            include_examples 'config.yml', '%<relative_yml_file_path>s'
+          end
+        RSPEC
+
+        attr_reader :feed_config
+
+        def initialize(_prompt, feed_config)
+          @feed_config = feed_config
+        end
+
+        def create
+          # TODO: ask for filename ... suggest defaults determined by on Helper.url_to(directory|file)_name
+          write_to_yml
+          scaffold_spec
+          print_files
+        end
+
+        def print_files
+          Helper.print_markdown <<~MARKDOWN
+            Created feed config at:
+              `#{files[:yml_file].gsub(files[:gem_root], '')[1..-1]}`
+
+            Created spec at:
+              `#{files[:spec_file].gsub(files[:gem_root], '')[1..-1]}`
+
+            Test with:
+              `bundle exec html2rss feed #{files[:yml_file].gsub(files[:gem_root], '')[1..-1]}`
```

Comment:
These two relative paths should directly come from files. No need for gsub stuff here.